.travis.yml

```diff
@@ -141,6 +141,11 @@ jobs:
       script:
         - scripts/travis/external_build.sh ./scripts/travis/deploy_packages.sh
 
+    - stage: post_deploy
+      os: linux
+      script:
+        - test/packages/test_release.sh
```

Comment:
Do we want all files living in scripts/travis/?

Reply:
We want all the invoked scripts by travis to live at `/scripts/travis`; that of course doesn't prevent you from creating a two-line bash script that invokes the `test/packages/test_release.sh`

Reply:
also, you want to add the 
```
      addons:
        apt:
          packages:
            - awscli
```
( or similar ) section to this so that it won't install all the list in the bottom of this file.

Reply:
I don't know if it's needed, but maybe there is a package named docker ?

Reply:
Given that you're still experimenting with this, please add the name also to
`allow_failures:` section, so that it won't cause any build failures.

Reply:
Done except for checking for a docker package.  Looking into it now.